lib/MigratorUtil.php

```diff
@@ -47,7 +47,9 @@ public static function getClassNameFromFile($file)
             for (;$i < count($tokens);$i++) {
                 if ($tokens[$i][0] === T_NAMESPACE) {
                     for ($j = $i + 1;$j < count($tokens); $j++) {
-                        if ($tokens[$j][0] === T_STRING) {
+                        if (\defined('T_NAME_QUALIFIED') && $tokens[$j][0] === T_NAME_QUALIFIED) {
+                            $namespace .= '\\' . $tokens[$j][1];
+                        } elseif ($tokens[$j][0] === T_STRING) {
```

Comment:
I think that's fine, I did a similar fix here https://github.com/phpbench/phpbench/pull/665/files#diff-6b0463b0983e774b73f65bfc37e8d70567eb9a7939e81e61976541661d8200e1R139